src/loki/reach.cc

```diff
@@ -5,34 +5,42 @@ using namespace valhalla::baldr;
 namespace valhalla {
 namespace loki {
 
-directed_reach SimpleReach(const DirectedEdge* edge,
-                           uint32_t max_reach,
-                           GraphReader& reader,
-                           const sif::EdgeFilter& edge_filter,
-                           const sif::NodeFilter& node_filter,
-                           uint8_t direction) {
-
-  // no reach to start with
+Reach::Reach() : Dijkstras() {
+  // Mock up the Location struct with the important stuff missing
+  auto* path_edge = locations_.Add()->add_path_edges();
+  path_edge->set_distance(0);
+  path_edge->set_begin_node(false);
+  path_edge->set_end_node(false);
```

Comment:
so here we prepopulate a location to do expansion from so we dont have to do this on every call to exact expansion